src/pages/PlanPage/subRoutes/CurrentOrgPlan/CurrentOrgPlan.tsx

```diff
@@ -35,13 +45,38 @@ function CurrentOrgPlan() {
       <InfoMessageStripeCallback />
       {accountDetails?.plan ? (
         <div className="flex flex-col gap-4 sm:mr-4 sm:flex-initial md:w-2/3 lg:w-3/4">
+          {planUpdatedNotification.alertOption ? (
+            <Alert
+              variant={
+                Object.values(AlertOptions).includes(
+                  planUpdatedNotification.alertOption as AlertOptions
```

Comment:
Is there a way to type this at the hook level so we don't need to cast here? may ease readability a bit too